src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap");
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+body {
+  --hover-active-color: #00acdc;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  padding: 22.5px 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  transition: all 0.3s;
+  height: 15px;
+  left: 509;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.moyo-logo {
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.is-active::after {
+  position: absolute;
+  display: block;
+  content: "";
+  width: 37px;
```

Don't hardcode width here.
```suggestion
  width: 100%;
```